libr/fs/meson.build

```diff
@@ -48,6 +48,16 @@ else
   ]
 endif
 
+if use_libsqsh
+  r_fs_sources += [
+    'p/fs_squashfs.c'
+  ]
+  r_fs_deps += [ libsqsh_dep ]
+  if get_option('blob')
+    r_fs_static_deps += [ libsqsh_static_dep ]
+  endif
```

Comment:
I think this should be removed too, shouldn't it?

Reply:
Its fine i think